persistence/src/main/scala/hmda/persistence/processing/HmdaFileParser.scala

```diff
@@ -122,6 +126,17 @@ class HmdaFileParser(submissionId: SubmissionId) extends HmdaPersistentActor {
     case GetState =>
       sender() ! state
 
+    case GetStatePaginated(page) =>
+      val tsErrState = state.tsParsingErrors
+      val tsErrorCount = if (tsErrState.isEmpty) 0 else 1
```

Comment:
This variable name took me a little while to understand.  From what I see, it's not actually the count of all the transmittal sheet errors, but rather it's counting the line as 1 if there are any errors.  Is that correct?  If so, maybe rename the variable to `tsLineError` or something similar, to make it clear you're counting the line itself.